provider/docker.go

```diff
@@ -463,6 +463,13 @@ func (provider *Docker) getSticky(container dockerData) string {
 	return "false"
 }
 
+func (provider *Docker) getIsBackendLBSwarm(container dockerData) string {
+	if _, err := getLabel(container, "traefik.backend.loadbalancer.swarm"); err == nil {
+		return "true"
```

Comment:
I'm not sure that's right here. It will always return true, whatever the value of the label is — and false otherwise 😅